pipeline_tools/tests/get_latest_schema_example_version.py

```diff
@@ -0,0 +1,48 @@
+import argparse
+import os
+
+class Version:
+    def __init__(self, dirname):
+        self.dirname = dirname
+        # Remove 'v' prefix
+        d = dirname[1:]
+        parts = d.strip().split('.')
+        numeric_parts = list(map(lambda x: int(x), parts))
+        self.v = numeric_parts
+        while len(self.v) < 3:  
+            self.v.append(0)
```

Comment:
This assumes the test(and our pipeline-tools) will only support major versions of the metadata schema, right? So each time the metadata team updated their minor versions(as well as patch versions), I think we should replace our directory, e.g. v5, with the new examples?

Reply:
Never mind, this will work for any versions.